defaults/test_default.yaml

```diff
@@ -118,6 +118,9 @@ append_scylla_args_oracle: '--enable-cache false'
 
 # cassandra-stress defaults
 stress_multiplier: 1
+stress_multiplier_w: 1
```

Comment:
I don't really like the kind of specific test parameters.

Those parameters won't work in all of the tests, and it's not clear to figure out which they would and in which they wouldn't.

I would think this is a per stress command parameter maybe we could add it to the stress command in a way we could extract it out of the command itself ? We had define something like that for the idea of having diffrent docker image per command, but it wasn't implemented.

Reply:
+1

in case we have `cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000` (the one you intent to change in this PR), you could for example add `processes=2` in the end, so inside the code that takes care of running the threads, you check that value, to use as a multiplier, and don't send it to the inner call of c-s, and then you have something that will work much easier to all c-s commands.. WDYT?

Reply:
I would call it `stress_multiplier` to avoid confusion, processes might be a bit vague in this regards.

Anyhow laying the foundations for having SCT parameters per command, would be helpful in more cases, like if we want one with `round_robin` and one without, defining which command is the main load and should be critical and which command shouldn't stop the test.

and I think we can have more examples like, that we were a bit limited by setting a "global' configuration option.

Reply:
May be then to split c-s parameters and and control parameters with -- 
`cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000 -- stress_multiplier=2 round_robin=True`

Reply:
> May be then to split c-s parameters and and control parameters with -- `cassandra-stress write no-warmup cl=QUORUM n=30000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -pop seq=1..30000000 -- stress_multiplier=2 round_robin=True`

yes, that is my suggestion

Reply:
Laying the foundations is good idea.
However, I wouldn't put it as suggested above.
We probably need to pick a better data structure, like nested yaml or list of lists or something like that...
E.g.
prepare_stress_cmd: [["c-s command", round_robin: True, stress_multiplier: 2],
                                      ["c-s command", round_robin: True, stress_multiplier: 3]]

Reply:
For now may we can use this implementation: https://github.com/scylladb/scylla-cluster-tests/pull/6293 and create task to refactore c-s commands config style?

Reply:
Yes, I agree.
I don't want to hold this PR for the refactor.